src/layouts/Login/LoginPage.tsx

```diff
@@ -30,7 +30,12 @@ import { useGetSgidAuth, useLogin, useVerifyOtp } from "hooks/loginHooks"
 import { getAxiosErrorMessage } from "utils/axios"
 import { useErrorToast, useSuccessToast } from "utils/toasts"
 
-import { IsomerLogo, LoginImage, OGPLogo, SingpassLogo } from "assets"
```

Comment:
if logos are not longer used, lets remove them!

Reply:
Removed OGPLogo in https://github.com/isomerpages/isomercms-frontend/pull/1619/commits/b29727ffa1d0655498c19b77918d97d316cf4d6b!